### ISSUE TITLE
In general, shortcuts should apply to all selected tabs

### ISSUE THREAD

**Issue by klazoklazo** [mbnuqw/sidebery#1750]
### Description

Default tab-related shortcuts (such as Ctrl+W for closing tabs and Ctrl+R for reloading tabs) should apply to all selected tabs in Sidebery similar to how it applies to all selected tabs in the horizontal tab bar.

Likewise, Sidebery-specific tab-related shortcuts (such as unloading all selected tabs) should apply to all selected in the horizontal tab bar similar to how it applies to all selected tabs in Sidebery.

Demo video:

https://github.com/user-attachments/assets/34c9c323-78b7-49c8-a909-ea3e690b73d4

**Comment by Awesome-E** [comment]
I agree. There should at least be an option to sync the selection state of tabs in Sidebery and tabs in the native tab bar. That way, that applying either a Sidebery shortcut or a native shortcut will apply to the same (selected) tabs.
